jsk_topic_tools/CMakeLists.txt

```diff
@@ -125,6 +125,16 @@ if (CATKIN_ENABLE_TESTING)
   target_link_libraries(test_rosparam_utils ${PROJECT_NAME})
   add_rostest_gtest(test_log_utils test/test_log_utils.launch src/tests/test_log_utils.cpp)
   target_link_libraries(test_log_utils ${PROJECT_NAME})
+
+  add_library(jsk_topic_tools_test
+    src/tests/test_nodelet_log_utils.cpp
+    src/log_utils.cpp)
+  target_link_libraries(jsk_topic_tools_test
+    ${catkin_LIBRARIES} ${BOOST_LIBRARIES})
+  add_executable(test_nodelet_log EXCLUDE_FROM_ALL test/test_nodelet_log.cpp)
+  target_link_libraries(test_nodelet_log ${catkin_LIBRARIES} ${GTEST_LIBRARIES})
+  add_dependencies(tests test_nodelet_log)
+  add_rostest(test/test_nodelet_log.test)
```

Comment:
I suppose you can do like this:
https://github.com/jsk-ros-pkg/jsk_common/blob/master/jsk_topic_tools/CMakeLists.txt#L126-L127